packages/types/src/utils/nanoid.ts

```diff
@@ -0,0 +1,25 @@
+import { nanoid } from 'nanoid'
+import { z } from 'zod'
+
+/** Generates a default value */
+export const nanoidSchema = z
+  .string()
+  .min(7)
+  .optional()
+  .default(() => nanoid())
```

Comment:
there’s also `z.nanoid()`, would that work, too?

Reply:
yea lets try it!

Reply:
alright I looked into it. Found two issues:
- we don't always use a nanoid so things like `default` will fail validation
- I couldn't find a way to generate a default using only zod

we can re-visit it in the future if we move away from the default uid